cloud-chatops/src/read_data.py

```diff
@@ -31,9 +38,9 @@ def validate_required_files() -> None:
     """
     This function checks that all required files have data in them before the application runs.
     """
-    repos = get_repos()
+    repos = get_config("repos")
     if not repos:
-        raise RepositoriesNotGiven("repos.csv does not contain any repositories.")
+        raise RepositoriesNotGiven("config.yml does not contain any repositories.")
```

Comment:
We don't have any docs for how the config.yml should look in this PR? It requires someone to effectively dig the source code

Reply:
Those changes are in #57 once it is merged and this one rebased then it will make sense.